src/main/kotlin/org/kamiblue/client/gui/rgui/component/Slider.kt

```diff
@@ -164,4 +168,6 @@ open class Slider(
         MouseState.HOVER -> GuiColors.hover
         else -> GuiColors.click
     }
+
+    open fun isBold() = false
```

Comment:
This can be a getter instead of a method and should be placed at top part of the class, after the listening field. Same in the overrides.